packages/react-tabs/src/components/Tab/__snapshots__/Tab.test.tsx.snap

```diff
@@ -2,26 +2,30 @@
 
 exports[`Tab renders correctly 1`] = `
 <div>
-  <div
+  <button
     class="fui-Tab"
     role="tab"
+    style="--fui-Tab__indicator--startOffset: 0px; --fui-Tab__indicator--endOffset: 0px;"
     tabindex="0"
```

Comment:
curious if we really need explicitly render button with tabindex, as that's the default value